src/main/kotlin/no/nav/tsm/mottak/dev_tools/ui/Index.kt

```diff
@@ -63,10 +69,14 @@ fun Routing.indexPageRoute(kafkaProducer: KafkaProducer<String, String>) {
         val someNumber = (100..200).random()
         kafkaProducer.send(
             ProducerRecord(
-                "sykmelding-utfall",
+                topic,
```

Comment:
korleis styrer du topic på L48 og her på L72 frå å være samme topic? som local dev er jo dette to forskjellige.

Reply:
nå er det bare et topic